tests/test_reconcile.py

```diff
@@ -31,7 +31,7 @@ def test_merge(self):
         msdf2 = parse_sssom_table(data_dir / "basic2.tsv")
         merged_msdf = merge_msdf(msdf1, msdf2)
 
-        self.assertEqual(71, len(merged_msdf.df))
+        self.assertEqual(24, len(merged_msdf.df))
```

Comment:
Why this change?

Reply:
with `match_type` changing to be single valued, duplicate rows get filtered out. So length o dataframes will change.